dev/set_matrix.py

```diff
@@ -566,6 +567,10 @@ def expand_config(config: dict[str, Any], *, is_ref: bool = False) -> set[Matrix
             )
             versions = get_latest_micro_versions(versions)
 
+            # Test every n minor versions if specified
+            if cfg.test_every_n_versions > 1:
+                versions = versions[:: -cfg.test_every_n_versions][::-1]
```

Comment:
Does this always contain the latest version?

Reply:
Ah it seems the list is not sorted. It always contain the latest version only if it is sorted.